src/System.Collections.Concurrent/ref/System.Collections.Concurrent.builds

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Collections.Concurrent.csproj" />
+    <Project Include="System.Collections.Concurrent.csproj">
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
Could you please make netcoreapp1.1 the default? that wold mean changing the .csproj to have netcoreapp1.1 as the nuget target moniker by default, and in here you will be cross compiling for netstandard1.3 instead. The reason to do this is because you want the default build of the .csproj to be the newest version of it.